### ISSUE TITLE
Joystick Mapping

### ISSUE THREAD

**Issue by whenwherewhy** [Improbable-AI/walk-these-ways#84]
We noticed that the joystick mapping in the picture is different from the one in the program, with button A corresponding to trotting. Is that true? 
Program: go1_gym_deploy/unitree_legged_sdk_bin/lcm_position.cpp # line148~164
              go1_gym_deploy/utils/cheetah_state_estimator.py # line 193

**Comment by whenwherewhy** [comment]
By the way, after pressing the R2 key twice, the robot enters the RL control stage. At this time, sometimes the robot will have a small amplitude of lateral drift. Is this a normal phenomenon

**Comment by gmargo11** [comment]
@whenwherewhy Yes, a small amount of lateral drift is normal for this controller.
